src/App.js

```diff
@@ -1,25 +1,87 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from 'react';
+import { nanoid } from 'nanoid';
+// import { localItems } from './db';
+import './styles.css';
+import ListItem from './components/ListItem';
+import Header from './components/Header';
+import AddItem from './components/AddItem';
+import Subheader from './components/Subheader';
+import Fetch from './components/Fetch';
+import ToggleButton from './components/Toggle';
 
 function App() {
+  // const [articles, setArticles] = useState(localItems);
+  const [articles, setArticles] = useState(loadFromLocal('products') ?? []);
+  const [searchValue, setSearchValue] = useState('');
+
+  const [language, setLanguage] = ToggleButton();
```

Comment:
Only components start with a capital letter. `ToggleButton` is a custom toggle hook, so I would recommend naming it `useToggle`. Everyone will then know, that it is a custom hook. :) 
Also custom hooks should not be in your `components` folder but in another folder: `hooks`